### PR TITLE
JS: import well-known types from google-protobuf package.

### PR DESCRIPTION
**Pull request by haberman**

### CODE REVIEW

File: js/gulpfile.js [hunk 1]
```diff
@@ -8,6 +8,20 @@ function exec(command, cb) {
 
 var protoc = process.env.PROTOC || '../src/protoc';
 
+var wellKnownTypes = [
```
Comment (murgatroid99) [inline js/gulpfile.js]: What about `google/protobuf/compiler/plugin.proto`? It looks like that's also on the list of well known types.

Comment (haberman) [inline js/gulpfile.js]: Done.

**Comment by murgatroid99** [comment]
LGTM

**Comment by jtattermusch** [comment]
@haberman  we also need also need a new version of the npm package (https://www.npmjs.com/package/google-protobuf) to be published to be able to benefit from this fix, right?  It there an ETA for the package being uploaded?

(CC @dgquintas  because he seems to be needing that to fix stuff in https://github.com/grpc/grpc/pull/6839).

**Comment by dgquintas** [comment]
ping on the npm package ETA :) It's blocking https://github.com/grpc/grpc/pull/6839 (not a critical PR, but it's been outstanding for a while).

**Pull request merged**